CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 ### 🐞 Fixed
 - Fixed user avatar in navigation drawer of the sample app. [#4050](https://github.com/GetStream/stream-chat-android/pull/4050)
 - The commands button in `MessageComposerView` can now be used to hide the command suggestion popup. [#4041](https://github.com/GetStream/stream-chat-android/pull/4041)
+- Now disabling "Quotes" option in the dashboard hides the "Reply" option in the message list. [#4074](https://github.com/GetStream/stream-chat-android/pull/4074)
```

Comment:
Maybe we could add the fact that `Threads & Replies` controls only threads now.
I like being very explicit.